cdap-app-templates/cdap-etl/cdap-etl-batch/src/main/java/co/cask/cdap/etl/batch/mapreduce/ETLMapReduce.java

```diff
@@ -202,7 +202,6 @@ public void initialize() throws Exception {
     for (String sourceName : phaseSpec.getPhase().getSources()) {
       try {
         BatchConfigurable<BatchSourceContext> batchSource = pluginInstantiator.newPluginInstance(sourceName, evaluator);
-
```

Comment:
It was removed on release/4.2:
https://github.com/caskdata/cdap/blob/release/4.2/cdap-app-templates/cdap-etl/cdap-etl-batch/src/main/java/co/cask/cdap/etl/batch/mapreduce/ETLMapReduce.java?utf8=%E2%9C%93#L204